src/http/api/profile.ts

```diff
@@ -374,6 +375,7 @@ export function createGetProfileFollowersHandler(
         };
 
         let page: CollectionPage | null = null;
+        let followers: Collection | null = null;
```

Comment:
We don't _need_ to have a shared `followers` variable here, we can just `await actor.getFollowers()` again in the second block - fedify won't do two lookups.

It's not a big deal, just letting you know ☺️